src/Marten/Events/Aggregation/EventListExtensions.cs

```diff
@@ -6,15 +6,20 @@ namespace Marten.Events.Aggregation;
 
 public static class EventListExtensions
 {
-    public static void FanOut<TSource, TChild>(this List<IEvent> events,
+    public static void FanOutEventData<TSource, TChild>(this List<IEvent> events,
```

Comment:
Hey, you can't make breaking API changes. Can you move this back to what it was before?

Reply:
@jeremydmiller, it is not a breaking change but a new method, have a look below; GitHub is showing the diff weirdly.

Reply:
Ok, my bad. This is a breaking change, as the fanout function has changed in the function below. @erdtsieck you need to keep naming the same, the new FanOut need to have a different name.

Reply:
I think it's non-breaking now